data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,9 @@
+export default function countValues(stack) {
+  let data = stack.pop();
+  let count = 0;
+  while (data !== undefined) {
+    count++;
+    data = stack.pop();
```

Comment:
Since you never examine `data` except in the `while` check, you could just move the pop into the `while` check.